cityofaustin/atd-moped
# Implement project status badge #7618

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/7618
Closes https://github.com/cityofaustin/atd-data-tech/issues/7640
Closes https://github.com/cityofaustin/atd-data-tech/issues/7639

## Testing
https://moped-test.austinmobility.io/moped

**Steps to test:**
- Create a new project, verify that it gets a "Potential" badge  (from #7635)
- Update project phase, verify that status badge updates
- Cancel project, verify that it no longer shows in project listing #7640

---
#### Ship list


## Preview(s):

https://user-images.githubusercontent.com/5282430/143905318-12ae4df2-0bdf-435f-98c0-0582bd77fa2e.mp4


https://user-images.githubusercontent.com/5282430/143905335-1fac59b1-5044-42b1-a57d-4ab0a7a5aab2.mp4


https://user-images.githubusercontent.com/5282430/143905347-76d853b8-9f94-4322-a483-b494466dec59.mp4


https://user-images.githubusercontent.com/5282430/143905359-015c8fe6-060f-41b7-91ce-0fd4c76ded7c.mp4
